ui/src/app/shared/workflow/wizard/node-add/node.wizard.component.ts

```diff
@@ -170,7 +177,9 @@ export class WorkflowNodeAddWizardComponent implements OnInit {
       projectKey: this.project.key,
       pipeline: this.newPipeline
     })).pipe(
-      finalize(() => this.loadingCreatePipeline = false),
+      finalize(() => {
+          this.loadingCreatePipeline = false;
```

Comment:
no need for markForCheck here ?

Reply:
no need because view is not updating after this call